geonode_mapstore_client/client/static/mapstore/configs/localConfig.json

```diff
@@ -2174,7 +2178,10 @@
                 "name": "Playback"
             },
             {
-                "name": "LayerDownload"
+                "name": "LayerDownload",
+                "cfg": {
+                    "disablePluginIf": "{!state('selectedLayerPermissions').includes('download_resourcebase')}"
```

Comment:
There are other `LayerDownload` configuration in localConfig all should include this disablePluginIf expression